examples/lavalink-basic-bot.rs

```diff
@@ -53,6 +57,7 @@ async fn main() -> anyhow::Result<()> {
         let http = HttpClient::new(token.clone());
         let user_id = http.current_user().await?.model().await?.id;
 
+        // The client is [`Lavalink`](crate::client::Lavalink) that forwards the required events from Discord.
```

Comment:
It's nice that you are trying to document what is going on here, but this is not a doc comment and won't get rendered anywhere. I'd try and rephrase it as a comment rather than documentation